lib/srv/sshserver.go

```diff
@@ -307,6 +317,12 @@ func (s *Server) getInfo() services.Server {
 func (s *Server) registerServer() error {
 	srv := s.getInfo()
 	if !s.proxyMode {
+		if s.injector != nil {
+			err := s.injector.Inject(srv)
```

Comment:
ouch! isn't this too low-level? this is "generic SSH guts" which I always appreciated. maybe somewhere higher up the stack in `services.go` would be better, no?